.yamato/project.metafile

```diff
@@ -8,16 +8,19 @@ test_platforms:
     image: package-ci/win10:stable
     flavor: b1.large
     editorpath: .Editor\Unity.exe
+    utr: .\utr.bat
   - name: mac
     type: Unity::VM::osx
     image: package-ci/mac:stable
-    flavor: m1.mac
+    flavor: b1.xlarge
```

Comment:
`large` as others vs `xlarge`?

Reply:
Good point, initially I thought that the xlarge would solve the startup timing issues but that was not the case at all, I shall revert this.

Reply:
Please take a look at [this change](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1195/commits/c8004bed8f4b89feec07653aa72b9be94ef997dd) and mark conversation as resolved if you are okay with it.